pkgs/tools/system/thermald/default.nix

```diff
@@ -33,10 +60,13 @@ stdenv.mkDerivation rec {
     cp ./data/thermal-conf.xml $out/etc/thermald/
   '';
 
+  outputDevdoc = "out";
```

Comment:
Could you keep it in a separate output (`outputs = [ "out" "devdoc" ];`) and also respect https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887?

Reply:
I hope I got the ordering right.